### ISSUE TITLE
HA error/info - discovery_info.ip

### ISSUE THREAD

**Issue by giovanne123** [JurajNyiri/HomeAssistant-Tapo-Control#127]
HomeAssistant - Tapo: Cameras Control: 3.4.3
Homeassistent 2021.12.2 reports in log:

Logger: homeassistant.helpers.frame
Source: helpers/frame.py:103
First occurred: December 16, 2021, 9:43:21 PM (1 occurrences)
Last logged: December 16, 2021, 9:43:21 PM

Detected integration that accessed discovery_info['ip'] instead of discovery_info.ip; this will fail in version 2022.6. Please report issue to the custom component author for tapo_control using this method at custom_components/tapo_control/config_flow.py, line 40: if self._async_host_already_configured(dhcp_discovery[IP_ADDRESS]):

**Comment by adekloet** [comment]
have got it to: 2021-12-17 10:20:47 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that accessed discovery_info['ip'] instead of discovery_info.ip; this will fail in version 2022.6. Please report issue to the custom component author for tapo_control using this method at custom_components/tapo_control/config_flow.py, line 40: if self._async_host_already_configured(dhcp_discovery[IP_ADDRESS]):

**Comment by viciovb** [comment]
I have the same issue. I guess the author needs to update the integration before HA 2022.6 is released.

**Comment by pr0fetul** [comment]
Hi,
Same here:
```
Detected integration that accessed discovery_info['ip'] instead of discovery_info.ip; this will fail in version 2022.6. Please report issue to the custom component author for tapo_control using this method at custom_components/tapo_control/config_flow.py, line 40: if self._async_host_already_configured(dhcp_discovery[IP_ADDRESS]):
```

**Comment by JurajNyiri** [comment]
Fixed in [3.4.4](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/3.4.4).
